avocado/plugins/spawners/podman.py

```diff
@@ -79,6 +79,10 @@ class PodmanSpawner(DeploymentSpawner, SpawnerMixin):
     description = 'Podman (container) based spawner'
     METHODS = [SpawnMethod.STANDALONE_EXECUTABLE]
 
+    def __init__(self, config=None):
+        super().__init__(config)
+        self.output_dir = None
```

Comment:
I am not sure what you mean by SpawnerMixin is already doing this. Because the `output_dir` is 
```
output_dir_path = os.path.join(self.job_output_dir, task.identifier.str_filesystem)
```
and the SpawnerMixin has only `job_output_dir`. But yes, I don't need to save that value and I compute it.

Reply:
Ohhh output_dir != job_output_dir. Do we really need both inside the Spawner?

Reply:
Yes you are right, base on your comment I decided to remove `output_dir` from the spawner, because it is Task specific and not Spawner specific variable.